readme.md

```diff
@@ -264,6 +264,27 @@ Now try some other options, e.g.
 0x -t examples/rest-api
 ```
 
+### Or using [Docker](https://www.docker.com/) and [Docker Compose](https://docs.docker.com/compose/)
+_this process has been tested on linux, different milage may apply on OSX with boot2docker_
+Clone this repo and `cd` to the `examples/docker` folder
+
+```
+docker-compose up
+```
+open another shell and send you SIGINT:
+
+```
+sudo kill -s SIGINT `ps aux | grep 'node /usr/local/bin/0x' | head -1| awk '{print $2}'`
```

Comment:
well, is not that "on osx doesn't happen". It's just that they stay inside the vm :)